input.rb

```diff
@@ -0,0 +1,19 @@
+class Input
+  def read_input
+    @input_store = []
+    @for_quit
+    begin
+      puts "Enter the values"
+      inputs = gets.chomp
+      if ( inputs.split(/ /).first.to_f.negative?() || inputs.split(/ /).last.to_f.negative?() )
```

Comment:
1. We can store `inputs.split(/ /)` in a variable.
2. No need to add empty `()` with method call.
```suggestion
      if ( inputs.split(/ /).first.to_f.negative? || inputs.split(/ /).last.to_f.negative? )
```